src/Thermal/HeatTransfer/sensors.jl

```diff
@@ -1,36 +1,57 @@
+"""
+Absolute temperature sensor in Kelvin.
```

Comment:
Names of units should be written in lowercase, e..g, kelvin.  See for example https://cdn.journals.aps.org/files/styleguide-pr.pdf sectionj III B.